optimum_benchmark/benchmarks/energy_star/config.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass, field
+from logging import getLogger
+from typing import Any, Dict, Union
+
+from ...system_utils import is_rocm_system
+from ..config import BenchmarkConfig
+
+LOGGER = getLogger("energy_star")
+
+
+@dataclass
+class EnergyStarConfig(BenchmarkConfig):
+    name: str = "energy_star"
+    _target_: str = "optimum_benchmark.benchmarks.energy_star.benchmark.EnergyStarBenchmark"
+
+    # dataset options
+    dataset_name: str = field(default="", metadata={"help": "Name of the dataset on the HF Hub."})
+    dataset_config: str = field(default="", metadata={"help": "Name of the config of the dataset."})
+    dataset_split: str = field(default="train", metadata={"help": "Dataset split to use."})
+    num_samples: int = field(default=-1, metadata={"help": "Number of samples to select in the dataset. -1 means all."})
+    input_shapes: Dict[str, Any] = field(
```

Comment:
it makes sense to merge this with a default dictionary (batch size: 1) to reduce user input
https://github.com/huggingface/optimum-benchmark/blob/92cf80f5cc3ec0799f48bd4495a604e47ca564d5/optimum_benchmark/benchmarks/inference/config.py#L51